src/interface.vue

```diff
@@ -70,10 +70,16 @@ export default defineComponent({
 		};
 
 		function compute() {
-			return props.template.replace(/{{.*?}}/g, (match) => {
+			const computedValue = props.template.replace(/{{.*?}}/g, (match) => {
 				const expression = match.slice(2, -2).trim();
 				return parseExpression(expression, values.value);
 			});
+
+			if (!computedValue) {
```

Comment:
If `computedValue` is a valid falsy value (0, false), it would still return `null`. That is not intended in some cases such as subtract or boolean expression.